docs/components/movement-sensor/mpu6050.md

```diff
@@ -81,10 +108,11 @@ Edit and fill in the attributes as applicable.
 
 ## Attributes
 
-| Name                  | Type    | Inclusion    | Description                                                                                                                                                                  |
-| --------------------- | ------- | ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `board`               | string  | **Required** | The `name` of the [board](/components/board/) to which the device is wired.                                                                                                  |
-| `i2c_bus`             | string  | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on your [board](/components/board/) wired to this device.                                         |
-| `use_alt_i2c_address` | boolean | **Required** | Depends on whether you wire AD0 low (leaving the default address of 0x68) or high (making the address 0x69). If high, set `true`. If low, set `false`. <br> Default: `false` |
+<!-- prettier-ignore -->
+| Name                  | Type    | Inclusion    | Description |
+| --------------------- | ------- | ------------ | ----------- |
+| `board`               | string  | **Required** | The `name` of the [board](/components/board/) to which the device is wired. |
+| `i2c_bus`             | string  | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on your [board](/components/board/) wired to this device. |
+| `use_alt_i2c_address` | boolean | Optional     | Depends on whether you wire AD0 low (leaving the default address of 0x68) or high (making the address 0x69). If high, set `true`. If low, set `false`. <br> Default: `false` |
```

Comment:
```suggestion
| `use_alt_i2c_address` | boolean | Optional     | Depends on whether you wire AD0 low (leaving the default address of `0x68`) or high (making the address `0x69`). If high, set `true`. If low, set `false`. <br> Default: `false` |
```

Although I could see monospace these values detracting from the importance of the monospaced booleans -- which are the actual toggles here. Up to you if you think this is too much monospace.

Reply:
Mm interesting point but I think it'd detract so will leave as-is.